restic/genericCommand.go

```diff
@@ -115,7 +116,7 @@ func (g *genericCommand) exec(args []string, options commandOptions) {
 	}
 }
 
-func (g *genericCommand) collectOutput(output io.ReadCloser, print bool) ([]string, error) {
+func (g *genericCommand) collectOutput(output io.ReadCloser, print bool, out chan string) ([]string, error) {
```

Comment:
`output.Close()` is never called inside this function. Having it as type `io.Reader` is sufficient.

Reply:
The boolean `print` smells.

Reply:
Changed it to `io.Reader` created an internal ticket to get rid of the smelly code.